zerver/actions/realm_settings.py

```diff
@@ -156,6 +157,67 @@ def do_set_realm_authentication_methods(
     )
     send_event(realm, event, active_user_ids(realm.id))
 
+# ! Added action function [commit -5]
+
+def do_set_realm_direct_message(
```

Comment:
Rather than writing a custom function for just this pair of settings, please just extend `do_set_realm_property` to support these group IDs. You'll see it uses a bunch of clever techniques to avoid needing to write custom code for each individual setting.